experiment/simulation/js/fibonacci.js

```diff
@@ -0,0 +1,346 @@
+const languages = ['py', 'js', 'c'];
```

Comment:
Same comments on this file. I see very little code that is different from the other 2 files. Please move it to a utility file and include it everywhere